src/components/PageButton.ts

```diff
@@ -0,0 +1,76 @@
+import { SFC, createElement } from "react";
+import * as classNames from "classnames";
+
+import { ButtonType, IconType } from "../utils/ContainerUtils";
+
+export interface PageButtonProps {
+    buttonType?: ButtonType;
+    showIcon?: IconType;
+    onClickAction?: () => void;
+    isDisabled?: boolean;
+    message?: string;
+}
+
+export const PageButton: SFC<PageButtonProps> = (props) => {
+    let iconClass = "";
+    let cssClass = "";
+    const disabledClass = { disabled: props.isDisabled };
+    const onClick = !props.isDisabled ? props.onClickAction : () => { return; };
+
+    if (props.buttonType === "firstButton") {
+        cssClass = "btn mx-button mx-name-paging-first";
+        iconClass = "glyphicon glyphicon-step-backward";
+    }
+
+    if (props.buttonType === "previousButton") {
```

Comment:
``` ts
} else if {
```

Reply:
@Andries-Smit  I did not put else if because readability becomes compromised

Reply:
Its okay i am modifying only on line 25

Reply:
Made it to look like this; Much easier to read

```
    if (props.buttonType === "firstButton") {

        cssClass = "btn mx-button mx-name-paging-first";
        iconClass = "glyphicon glyphicon-step-backward";

    } else if (props.buttonType === "previousButton") {

        cssClass = "btn mx-button mx-name-paging-previous";
        iconClass = "glyphicon glyphicon-backward";

    } else if (props.buttonType === "nextButton") {

        cssClass = "btn mx-button mx-name-paging-next";
        iconClass = "glyphicon glyphicon-forward";

    } else if (props.buttonType === "lastButton") {

        cssClass = "btn mx-button mx-name-paging-last";
        iconClass = "glyphicon glyphicon-step-forward";
    }
```